app/react/Templates/specs/utils/renderConnected.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureStore, { MockStore, MockStoreCreator } from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+
+const middlewares = [thunk];
+const mockStoreCreator: MockStoreCreator<object> = configureStore<object>(middlewares);
+
+const renderConnected = (
+  Component: React.ReactType,
+  props: object,
+  storeData: object = {
+    template: {
+      data: { properties: [], commonProperties: [] },
+      formState: {
+        fields: [],
+        $form: {
+          errors: {},
+        },
+      },
+    },
+  }
+) => {
+  const store: MockStore = mockStoreCreator(storeData);
+  return shallow(
+    <Provider store={store}>
+      <Component {...props} />
+    </Provider>
+  )
+    .dive({ context: { store } })
+    .dive();
+};
+
+export { renderConnected };
```

Comment:
True, we should use this technique to test connected components instead of exporting the class by itself on new components and refactors, we can move this file later to a more generic folder.